src/dice.js

```diff
@@ -1,71 +1,84 @@
-dice = {
-	version: "0.7.0",
+var dice = {};
+var parse = require('./parser.js');
+var eval = require('./evaluate.js');
+dice.parse = parse.parse;
+dice.eval = eval.eval;
+dice.version = "0.8.0";
 
-	roll: function(str, scope){
-		var parsed = dice.parse.parse(str);
-		var evaled = dice.eval.eval(parsed, scope);
-		return evaled;
-	},
+function roll(str, scope){
+	var parsed = parse.parse(str);
+	var evaled = eval.eval(parsed, scope);
+	return evaled;
+};
 
-	statistics: function(str, scope, samples){
-		if(typeof(scope) == "number"){
-			samples = scope;
-			scope = {};
-		}
-		scope = scope || {};
-		samples = samples || 1000;
-		var resultSet = [];
-		var i;
-		for(i = 0; i < samples; i++){
-			resultSet.push(dice.roll(str, scope));
-		}
-		var mean = resultSet.reduce(function(n, acc){ return n + acc; }, 0) / samples;
-		var min = resultSet.reduce(function(n, acc){ return n < acc ? n : acc; }, resultSet[0]);
-		var max = resultSet.reduce(function(n, acc){ return n > acc ? n : acc; }, resultSet[0]);
-		return {
-			'results': resultSet,
-			'mean': mean,
-			'min': parseInt(min.toFixed()),
-			'max': parseInt(max.toFixed())
-		};
-	},
+dice.roll = roll;
```

Comment:
Mostly a style thing, but is there any reason not to do:

```javascript
dice.roll = function(...) { ... }
```

Less repetition, mostly.